models/task_to_create.go

```diff
@@ -0,0 +1,7 @@
+package models
+
+// TaskToCreate is a type that contains the details required for creating a Task type.
+type TaskToCreate struct {
```

Comment:
Is there a reason this is split into its own file? Could it live with the other task models?

Reply:
I thought it was the convention but maybe not - I've moved it now